amical/_cli/commands/calibrate.py

```diff
@@ -0,0 +1,118 @@
+from glob import glob
+from pathlib import Path
+
+from astroquery.simbad import Simbad
+from matplotlib import pyplot as plt
+from tabulate import tabulate
+from termcolor import cprint
+
+import amical
+
+
+def _query_simbad(targetname):
+    """Check on Simbad if the target is supposed to be a calibrator or a science
+    source."""
+    customSimbad = Simbad()
+    customSimbad.add_votable_fields("otype")
+    res = customSimbad.query_object(targetname)
+    otype = res["OTYPE"][0]
+    if otype == "Star":
+        nrm_type = "CAL"
+    else:
+        nrm_type = "SCI"
+    return nrm_type
+
+
+def _select_association_file(args):
+    """Show report with the data found and allow to select the science target
+    (SCI) to be calibrated and the calibrator (CAL)."""
+    l_file = sorted(glob("%s/*.h5" % args.datadir))
+
+    if len(l_file) == 0:
+        raise OSError("No h5 files found in %s, check --datadir." % args.datadir)
+
+    index_file = []
+
+    headers = ["FILENAME", "TARGET", "DATE", "INSTRUM", "TYPE", "INDEX"]
+
+    d = []
+    for i, f in enumerate(l_file):
+        bs = amical.load_bs_hdf5(f)
+        filename = Path(f).stem
+        hdr = bs.infos.hdr
+        target = hdr.get("OBJECT", None)
+        date = hdr.get("DATE-OBS", None)
+        ins = hdr.get("INSTRUME", None)
+
+        if target not in (None, "Unknown"):
+            try:
+                source = _query_simbad(target)
+            except Exception:
+                source = "Unknown"
+        else:
+            target = "Unknown"
+            source = "Unknown"
+
+        d.append([filename, target, date, ins, source, i])
```

Comment:
I don't understand why you're constructing single-element list of lists, can you confirm that's the intent ?
It would help making the var name explicit

Reply:
I construct the list of information from multiple files (in l_file list) to be printed with tabulate.